content/source/docs/enterprise/before-installing/centos-requirements.html.md

```diff
@@ -10,22 +10,19 @@ When installing Terraform Enterprise on CentOS Linux, ensure your OS and Docker
 ## Install Requirements
 
 * A [supported version](/docs/enterprise/before-installing/index.html#operating-system-requirements) of CentOS.
-* A suitable version of Docker:
-   * Docker 1.13.1 (available in the `extras` repository).
-   * [Docker CE](https://docs.docker.com/install/linux/docker-ce/centos/) version 17.06 through 19.03.8.
-   * [Docker EE](https://docs.docker.com/install/linux/docker-ee/centos/) version 17.06 through 19.03.8.
-   * Or you can allow the installer to install Docker for you.
-* A properly configured docker storage backend, either:
-   * Devicemapper configured for production usage, in accordance with the [Docker documentation](https://docs.docker.com/storage/storagedriver/device-mapper-driver/#configure-direct-lvm-mode-for-production). This configuration requires a second block device available to the system to be used as a thin-pool for Docker. You may need to configure this block device before the host system is booted, depending on the hosting platform.
-**Note:** Current, [Docker documentation] (https://docs.docker.com/storage/storagedriver/select-storage-driver/) states that the `devicemapper` storage driver is deprecated in Docker Engine 18.09 and will be removed in a future release. It is recommended that users of the `devicemapper` storage driver migrate to `overlay2`. 
-   
-   * A system capable of using overlay2. The requires at least kernel version 3.10.0-693 and, if XFS is being used, the flag `ftype=1`. The full documentation on this configuration is on the [Docker site](https://docs.docker.com/storage/storagedriver/overlayfs-driver/).
-
-If you choose to have Docker installed via the install script, ensure that `/etc/docker/daemon.json` is set up correctly, first.  The installer's default configuration sets up the devicemapper driver to use a loopback file, which is explicitly not supported, and the installation script will fail.  Setting up the driver for direct-lvm usage before installation will help ensure a successful installation.
+* One of the following installations of Docker:
+  * Docker CE 17.06 or later. Docker CE can either be pre-installed by the operator or installed via our installation script. If Docker CE is pre-installed by the operator, be sure to pass the `no-docker` flag to the installation script to prevent it from trying to install Docker CE again.
+  * Docker EE 17.06 or later. Details for installing Docker EE can be found [here](https://docs.mirantis.com/containers/v3.1/mcr-deployment-guide/mcr-linux/rhel.html).
```

Comment:
I'm of the opinion we shouldn't link to instructions for this, since there aren't Docker.com or RedHat.com instructions.

Reply:
I'm with it. Between Docker, Moby, and Mirantis I'm not sure what to believe anymore. Making the change now.